e2e/cypress/integration/ShowList.js

```diff
@@ -0,0 +1,42 @@
+/// <reference types="Cypress" />
+
+describe("a user visiting the show list page", () => {
+  beforeEach(() => {
+    cy.task("db:truncate", "Show")
+    cy.task("db:insert", { modelName: "Show", json: {name: "Supernatural", description: "Brothers fighting monsters"}})
+    cy.task("db:insert", { modelName: "Show", json: {name: "Friends", description: "Friends will be there for you"}})
```

Comment:
this can be combined into one insert statement, you just need to designate `json: [{...object one}. {object.two}]`

Reply:
you can add a space after the end of a string at the end of lines 6 and 7